src/styles/card.scss

```
@@ -0,0 +1,126 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 15px;
+  font-family: $main-font;
+  font-weight: 400;
+}
+
+.card__picture {
+  height: 134px;
+}
+
```

```suggestion
.card {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 200px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  padding: 30px 14px 15px;
  font-family: $main-font;
  font-weight: 400;
  
  &__picture {
    height: 134px;
  }
}
```

Используй во всех файлах scss nesting